test/extensions/filters/http/transformation/inja_transformer_test.cc

```diff
@@ -637,6 +675,32 @@ TEST_F(InjaTransformerTest, DontParseBodyAndExtractFromIt) {
   EXPECT_EQ(body.toString(), "json");
 }
 
+TEST_F(InjaTransformerTest, DontParseBodyAndExtractFromReplacementText) {
+  Http::TestRequestHeaderMapImpl headers{{":method", "GET"}, {":path", "/foo"}};
+  Buffer::OwnedImpl body("not json body");
+
+  TransformationTemplate transformation;
+  transformation.set_parse_body_behavior(TransformationTemplate::DontParse);
+  transformation.set_advanced_templates(true);
+
+  envoy::api::v2::filter::http::Extraction extractor;
+  extractor.mutable_body();
+  extractor.set_regex("not ([\\-._[:alnum:]]+) body");
+  extractor.set_subgroup(1);
+  auto replacement_text = "JSON";
+  extractor.mutable_replacement_text()->set_value(replacement_text);
```

Comment:
nit: this var can be inlined

Reply:
Resolved in the replace tests here: e8f9e74fbd152723852eba90849fa6e58bed0daf

Just realized this comment was on tests added in inja_transformer_test.cc -- I'll put up a commit to fix this soon

Reply:
Resolved in inja_transformer_test.cc here: 0a7d48eefab514ce13debc4efd01c6fe0b4f32ce